cobbler/actions/reposync.py

```diff
@@ -575,30 +579,17 @@ def yum_sync(self, repo):
 
         repodata_path = os.path.join(dest_path, "repodata")
         repomd_path = os.path.join(repodata_path, "repomd.xml")
-
         if os.path.exists(repodata_path) and not os.path.isfile(repomd_path):
             shutil.rmtree(repodata_path, ignore_errors=False, onerror=None)
 
+        repodata_path = os.path.join(temp_path, "repodata")
+        if os.path.exists(repodata_path):
+            shutil.rmtree(repodata_path, ignore_errors=False, onerror=None)
```

Comment:
I don't know your opinion on this but I would print a message to the log that informs the user that we deleted his repodata. Otherwise this could be unexpected for some users.

Reply:
@SchoolGuy This is the main fix for this PR. librepo could not correctly update changed metadata in .origin/repodata and an error occurred. Although of course deletion is not required for a static repo. I find it easier to delete the metadata and redownload it than to try to determine if it has changed.

Reply:
@SchoolGuy I'll add a warning about deleting old metadata to the code.

Reply:
@tpw56j Sorry if my wording was not clear. I don't want to change this. I only want to add a `self.logger.info("Deleted repo metadata for %s", repodata_path)` or something like this.